hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-oadp.adoc

```diff
@@ -0,0 +1,85 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-disaster-recovery-oadp"]
+= Disaster recovery for a hosted cluster by using {oadp-short}
+include::_attributes/common-attributes.adoc[]
+:context: hcp-disaster-recovery-oadp
+
+toc::[]
+
+You can use the {oadp-short} Operator to perform disaster recovery on {aws-first} and bare metal.
+
+The disaster recovery process with {oadp-short} involves the following steps:
+
+. Preparing your platform, such as {aws-full} or bare metal to use {oadp-first}
+. Backing up the data plane workload
+. Backing up the control plane workload
+. Restoring a hosted cluster by using OADP
+
+[id="dr-oadp-prerequisites"]
+== Prerequisites
+
+You must meet the following prerequisites on the management cluster:
+
+* You xref:../../backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc#oadp-installing-operator[installed the {oadp-short} Operator].
+* You created a storage class.
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {oadp-short} subscription through a catalog source.
+* You have access to a cloud storage provider that is compatible with {oadp-short}, such as, S3, {azure-full}, {gcp-full}, or MinIO.
+* In a disconnected environment, you have access to a self-hosted storage provider, for example link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/[{odf-full}] or link:https://min.io/[MinIO] that is compatible with {oadp-short}.
+* Your hosted control planes pods are up and running.
+
+[id="hcp-dr-oadp-prepare-aws"]
+== Preparing {aws-short} to use {oadp-short}
+
+To perform disaster recovery for a hosted cluster, you can use {oadp-first} on {aws-first} S3 compatible storage. After creating the `DataProtectionApplication` object, new `velero` deployment and `node-agent` pods are created in the `openshift-adp` namespace.
+
+To prepare {aws-short} to use {oadp-short}, see "Configuring the {oadp-full} with Multicloud Object Gateway" in Additional resources.
```

Comment:
Per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations):
```suggestion
To prepare {aws-short} to use {oadp-short}, see "Configuring the {oadp-full} with Multicloud Object Gateway".
```